Houseclub/src/main/java/me/grishka/houseclub/fragments/ProfileFragment.java

```diff
@@ -85,10 +85,34 @@ public View onCreateContentView(LayoutInflater inflater, ViewGroup container, Bu
 		v.findViewById(R.id.inviter_btn).setOnClickListener(this::onInviterClick);
 		if(self)
 			bio.setOnClickListener(this::onBioClick);
+		photo.setOnClickListener(this::onAvatarClick);
```

Comment:
This would set the listener in others' profiles as well. You probably don't want that.